.github/workflows/dev_pr.yml

```diff
@@ -38,6 +38,7 @@ jobs:
           persist-credentials: false
 
       - name: Check title for Conventional Commits format
+        continue-on-error: true
```

Comment:
If we add this, this workflow may not report an error with unexpected title...

Reply:
It currently reports an error via a comment, though I'd like to change that (#1294). But you're right, it looks like the right way to do this is [`if: always()`](https://stackoverflow.com/questions/62045967/is-there-a-way-to-continue-on-error-while-still-getting-correct-feedback)